include/nlohmann/json.hpp

```diff
@@ -3215,18 +3306,25 @@ class basic_json
 
     @since version 1.0.0
     */
-    template < typename ValueType, typename std::enable_if <
-                   !std::is_pointer<ValueType>::value&&
-                   !std::is_same<ValueType, detail::json_ref<basic_json>>::value&&
-                   !std::is_same<ValueType, typename string_t::value_type>::value&&
-                   !detail::is_basic_json<ValueType>::value
-                   && !std::is_same<ValueType, std::initializer_list<typename string_t::value_type>>::value
-#if defined(JSON_HAS_CPP_17) && (defined(__GNUC__) || (defined(_MSC_VER) && _MSC_VER >= 1910 && _MSC_VER <= 1914))
-                   && !std::is_same<ValueType, typename std::string_view>::value
+    template <
+        typename ValueType,
+        typename std::enable_if <
+            !std::is_pointer<ValueType>::value&&
+            !std::is_same<ValueType, detail::json_ref<basic_json>>::value&&
+            !std::is_same<ValueType, typename string_t::value_type>::value&&
+            !detail::is_basic_json<ValueType>::value&&
+            !std::is_same <
+                ValueType,
+                std::initializer_list<typename string_t::value_type >>::value
+#if defined(JSON_HAS_CPP_17) && \
+    (defined(__GNUC__) ||       \
+     (defined(_MSC_VER) && _MSC_VER >= 1910 && _MSC_VER <= 1914))
+            && !std::is_same<ValueType, typename std::string_view>::value
 #endif
-                   && detail::is_detected<detail::get_template_function, const basic_json_t&, ValueType>::value
-                   , int >::type = 0 >
-    operator ValueType() const
+            && detail::is_detected<detail::get_template_function,
+                                   const basic_json_t&, ValueType>::value,
+            int >::type = 0 >
+    JSON_EXPLICIT operator ValueType() const
```

Comment:
As this is the only place where `JSON_EXPLICIT` is actually used, please revert all the whitespace/formatting changes to have a better overview what actually changed.